ballsdex/settings.py

```diff
@@ -139,6 +143,8 @@ def read_settings(path: "Path"):
     settings.max_attack_bonus = content.get("max-attack-bonus", 20)
     settings.max_health_bonus = content.get("max-health-bonus", 20)
 
+    settings.packages = content["packages"] or []
```

Comment:
Because of odd design, the content of the yaml is read before update, so you need to handle the case where the list doesn't exist yet
```suggestion
    settings.packages = content.get("packages") or [
        "ballsdex.packages.admin",
        "ballsdex.packages.balls",
        "ballsdex.packages.config",
        "ballsdex.packages.countryballs",
        "ballsdex.packages.info",
        "ballsdex.packages.players",
        "ballsdex.packages.trade",
    ]
```